src/recursive_max/__init__.py

```diff
@@ -0,0 +1,8 @@
+def recursive_max(arr):
+    assert arr, "List can not be empty"
+
+    head = arr[0]
+    if len(arr) == 1:
+        return head
+    x = recursive_max(arr[1:])
+    return head if head > x else x
```

Comment:
`head` & `x` is incoherent names, usually peoples use `head` & `tail` or `x` & `xs`

Reply:
I was inattentive, it isn't tail :( Sorry, man, name it `maxOther` or alike

Reply:
I've renamed it to `guess_max`